sim/hunter/marksmanship/chimera_shot.go

```diff
@@ -62,20 +58,18 @@ func (hunter *Hunter) registerChimeraShotSpell() {
 	})
 }
 
-func (hunter *Hunter) chimeraShotSerpentStingSpell() *core.Spell {
+func (hunter *MarksmanshipHunter) chimeraShotSerpentStingSpell() *core.Spell {
```

Comment:
I'm not sure how this spell works in cata so i might be wrong but isnt this supposed to do % of the total damage of Serpent Sting? You can check the SOD implementation for how we do it there

Reply:
Actually it doesnt in Cata, shouldnt be there at all, removing it.